unix/Makefile

```diff
@@ -74,10 +77,14 @@ ifeq ($(MICROPY_FORCE_32BIT),1)
 # starting with linux-libc-dev:i386
 ifeq ($(MICROPY_PY_FFI),1)
 ifeq ($(UNAME_S),Linux)
+ifeq ($(UNAME_M), armv7l)
+CFLAGS_MOD += -I/usr/include/arm-linux-gnueabihf
```

Comment:
The Raspberry Pi 3 just came out. It would be detected as a 64-bit platform and the build would break.
I any case, if you think it's confusing, I'll remove it from this PR.